saltpylint/__about__.py

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
+
+__title__ = 'SaltPyLint'
+__description__ = 'Required PyLint plugins needed in the several '\
+    'SaltStack projects.'
+
+__version_info__ = (2017, 5, 19)
+__version__ = '{0}.{1}.{2}'.format(*__version_info__)
```

Comment:
I did a simple bgoogle search regarding `__about__`, didn't find any occurrences so this also seems non-stardard. As a matter of personal preference, I like it how it was.